### ISSUE TITLE
Iphone Safari Browser shows changes only for a moment

### ISSUE THREAD

**Issue by franzel61** [rkopka42/Wasserwaage#6]
Hi rkopka42,

Thanks for this great project, I‘m facing the issue that only on the movement of the MPU9250 with Esp32 Wroom the changes are shown in the browser. At the next moment its even again

**Comment by rkopka42** [comment]
That is a combination I didn't try. Have you changed the defines to get the MPU9250 inserted without M5STACK or just used the M5STACK option ?
From the description it looks as if you are using the gyro instead of the accel values. I never had such an effect during development with either sensor. Did you use one of the debugging outputs of the accel values from raw to calculated values ?

**Comment by franzel61** [comment]
I use the ESP32 Wroom  with mpg-9250/6500 Tecnoiot, I didn't do any setting an USE_M5STACK LIB, no Monitor assigned just via web
I have done a small video it's on my one drive https://1drv.ms/v/s!AgrH9eJ2SUWvhOtGa8PJh4quOkK5DA?e=z9sDjr

**Comment by rkopka42** [comment]
Yes it looks like the gyro values. What did you change on the code ? As it is you can choose M5STACK with 9250 or no M5STACK with MMA8452Q. The later won't work at all with the 9250. With monitor I mean the serial monitor of the Arduino IDE. You can uncomment code to show the values used to calculate the angles.

**Comment by franzel61** [comment]
I didnt change the code, just compiled. do you recommend to use a other library as M5Stack? The board is named HW GY-6500 GY9250 

[200801_0718_WasserwaageLog.pdf](https://github.com/rkopka42/Wasserwaage/files/5011633/200801_0718_WasserwaageLog.pdf)

**Comment by rkopka42** [comment]
I think the problem is not the M5STACK as such. The SW has many options for different HW and usages. Most users will have to make the appropriate selections before compiling. As it is there are two main setups. M5Stack with 9250 and display and no M5STACK with the other chip and no display (which can be selected individually). Just compiling it will easily result in random effects. The code as it is is not intended for just getting and using it without further adjustments on all HW variants.

Your setup is not predefined. From the log I see very strange fcr and fcp values. These should be 0 from start. As they are changed you either calibrated from WEB or the button on pin 37 (M5STACK module button) was pressed. As you have probably not connected anything to this pin it calibrates continuously. Therefor each change in position is immediately compensated to zero.

Either put this pin to VCC (as it is checked as !input), which will probably not be possible, as GPIO37 isn't available on most ESP32 boards. But you could change the value in defines.h line56 and select a pin available.
Or comment the whole section in Wasserwaage.ino from line 493 to 560.
